Add improvements. (#2639)
wso2/docs-is: en/docs/references/concepts/authentication/discovery.md

 </tbody>
 </table>
 
 The following endpoint is responsible for revealing the OpenID Provider's
 issuer after validating the required parameters (Resource, Host and rel). 
 
 ``` https://<SERVER_URL>/.well-known/webfinger```
 
### Expose metadata
 
 The OpenID provider exposes a JSON document listing all the metadata of the server. 

>  This document contains of standard endpoints and other server supported metadata values. 

Reviewer: ```suggestion
 This document contains standard endpoints and other server supported metadata values. 
```

>  The target audience of this file is the client application developers who send requests to the

Reviewer: ```suggestion
 The target audience of this file are the client application developers who send requests to the
```

 server.
 
 The following endpoint is responsible for exposing the server's metadata.
 
 ```https://<SERVER_URL>/.well-known/openid-configuration```
 

>  A sample JSON file that will contain server metadata is shown below.

Reviewer: ```suggestion
 A sample JSON file that containing the server metadata is shown below.
```
